Query layout nav support. (#67)
Financial-Times/o-layout: MIGRATION.md

 </div>
```

> - Move the `o-layout__navigation` class from the list (`ul` or `ol`) to the containing `nav` element. This only applies if you have added navigation manually. If your project relies on `o-layout` JavaScript to generate a nav no action is required.

Reviewer: ```suggestion
- If you have added navigation manually, you'll need to move the `o-layout__navigation` class from the list (`ul` or `ol`) to the containing `nav` element. 

If your project relies on `o-layout` JavaScript to generate a nav, no action is required.
```
